src/main/java/org/opentripplanner/graph_builder/module/osm/OsmModule.java

```diff
@@ -55,11 +57,13 @@ public class OsmModule implements GraphBuilderModule {
   private final SafetyValueNormalizer normalizer;
   private final VertexGenerator vertexGenerator;
   private final OsmDatabase osmdb;
+  private final StreetLimitationParameters streetLimitationParameters;
 
   OsmModule(
     Collection<OsmProvider> providers,
     Graph graph,
     DataImportIssueStore issueStore,
+    @Nullable StreetLimitationParameters streetLimitationParameters,
```

Comment:
Under what circumstances can this be null?

Reply:
As I said in another comment, I would like this to be non-nullable and the builder should provide a default value.